pkg/kn/commands/source/apiserver/apiserver_flags.go

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2019 The Knative Authors
```

Comment:
can we move the flags to `pkg/kn/commands/flags/` ?

Reply:
These are flags for `kn source create apiserver` only.
I think it's better to keep it in `commands/source/apiserver` folder.

Reply:
It's more about having the kn flags together in flags package, right now not all commands flags are following it (only traffic flags). We've an open issue for this https://github.com/knative/client/issues/337.

Reply:
I have moved flags to that folder.